crisischeckin/Services/MessageCoordinator.cs

```diff
@@ -21,9 +21,10 @@ public void SendMessage(Message message, List<MessageRecipient> recipients)
                 {
                     messageSender.SendMessage(message, recipients);
                 }
-                catch (Exception)
+                catch (Exception e)
                 {
                     //TODO: Add logging implementation--awaiting decision from Bill
+                    ////throw e;
```

Comment:
We already have elmah as part of the project. Can you update this pull request to include the logging here? Thanks.